spec.html

```diff
@@ -42121,15 +42122,15 @@ <h2>Syntax</h2>
 
         FourToSeven :: one of
           `4` `5` `6` `7`
+
+        NonOctalDecimalEscapeSequence :: one of
+          `8` `9`
```

Comment:
When there's only 2 alternatives, it feels a bit unnecessary to use "one of".

Reply:
Hmm, that's a copy-paste from [B.1.1](https://tc39.es/ecma262/#sec-additional-syntax-numeric-literals), which in turns follows the suit of [11.8.3](https://tc39.es/ecma262/#sec-literals-numeric-literals) (e.g., ExponentIndicator and BinaryDigit).

We could change them all if we wanted to enforce a minimum of three disjuncts though.

Reply:
No it's probably not worth it.